src/Sentry/Platforms/Android/SentrySdk.cs

```diff
@@ -220,7 +220,9 @@ private static void AndroidEnvironment_UnhandledExceptionRaiser(object? _, Raise
             return null;
         }
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
What will be the trigger for when we remove these? When the offending members actually get removed from the dotnet framework?